### PR TITLE
allken_et_al_2012_cookbook

### PR DESCRIPTION
**Pull request by cedrict**
Pull Request Checklist. Please read and check each box with an X. Delete any part not applicable. Ask on the [forum](https://community.geodynamics.org/c/aspect) if you need help with any step.

*Describe what you did in this PR and why you did it.*

### Before your first pull request:

* [ x] I have read the guidelines in our [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md) document.

### For all pull requests:

* [ ] I have followed the [instructions for indenting my code](../blob/main/CONTRIBUTING.md#making-aspect-better).

### For new features/models or changes of existing features:

* [ ] I have tested my new feature locally to ensure it is correct.
* [ ] I have [created a testcase](http://www.math.clemson.edu/~heister/manual.pdf#sec%3Awriting_tests) for the new feature/benchmark in the [tests/](../blob/main/tests/) directory.
* [ ] I have added a changelog entry in the [doc/modules/changes](../blob/main/doc/modules/changes) directory that will inform other users of my change.

### CODE REVIEW

**Comment by jdannberg** [comment]
Thanks for contributing this! Before I look at the PR in more detail, one general comment: For the cookbook to be rendered in sphinx, you need to add it to one of the index files (in your case, it would probably be https://github.com/geodynamics/aspect/blob/main/doc/sphinx/user/cookbooks/geophysical-setups.md). Only once you do that, the cookbook will appear in the in the docs/readthedocs tester (so only then, I can actually see if the formatting works).

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 1]
```diff
@@ -0,0 +1,168 @@
+#  Global parameters
+set Dimension                              = 3
+set Start time                             = 0
+set End time                               = 5e6
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = iterated Stokes
+set Nonlinear solver tolerance             = 1e-3
+set Max nonlinear iterations               = 25
+set Output directory                       = output_allken_etal_2012
```
Comment (gassmoeller) [inline cookbooks/allken_et_al_2012_rift_interaction/allken.prm:9]:
```suggestion
set Output directory                       = output-allken_etal_2012
```

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+# Mesh refinement specifications.
+subsection Mesh refinement
+  set Initial adaptive refinement        = 2
+  set Initial global refinement          = 2
+  set Time steps between mesh refinement = 1
+  set Refinement fraction                = 0.5
+  set Strategy                           = strain rate
+  set Coarsening fraction                = 0
+  #set Run postprocessors on initial refinement = true
```
Comment (gassmoeller) [inline cookbooks/allken_et_al_2012_rift_interaction/allken.prm:41]:
```suggestion
```

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  set Additional tangential mesh velocity boundary indicators = left,right
+  subsection Free surface
+    set Surface velocity projection = normal
+  end
+end
+
+# Velocity boundary conditions
+# The imposed velocity produces a background strain-rate of 2e-15.
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = 2,3,4 #bottom,
```
Comment (gassmoeller) [inline cookbooks/allken_et_al_2012_rift_interaction/allken.prm:62]: Please use the names. I think this is `right, top, bottom`?

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+(sec:cookbooks:allken)=
+# rift interaction in brittle-ductile coupled systems
```
Comment (gassmoeller) [inline cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md:2]:
```suggestion
# Rift interaction in brittle-ductile coupled systems
```

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+*This section was contributed by Cedric Thieulot*
+
+The setup for this experiment is based of {cite:t}`alht11,alht12,alhf13`.
+The Cartesian domain is $L_x\times L_y\times L_z=\SI{210}{\km}\times\SI{210}{\km}\times\SI{30}{\km}$
+in size (see {numref}`fig:allken_setup`).
+It is a simple 2-layer crustal model consisting of $h_{uc}=\SI{15}{\km}$ of upper crust (characterised by a Mohr-Coulomb rheology with cohesion
```
Comment (gassmoeller) [inline cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md:9]: `\SI` doesnt work anymore, we replace it with the following: (check other places below).
```suggestion
It is a simple 2-layer crustal model consisting of $h_{uc}=15 \text{ km}$ of upper crust (characterised by a Mohr-Coulomb rheology with cohesion
```

**Comment by cedrict** [comment]
done :)

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+$C^0=\SI{20}{\mega\pascal}$ and angle of friction $\phi^0=\SI{16}{\degree}$) and $h_{lc}=\SI{15}{\km}$ of lower crust
+(characterised by a Newtonian viscous rheology with a viscosity $\eta_{lc}=\SI{1e20}{\pascal\second}$).
+The flow is assumed to be incompressible and isothermal.
+Both material have the same density $\rho=\SI{2800}{\kg\per\cubic\meter}$ and the gravity is vertical with magnitude \SI{9.81}{\meter\per\square\second}.
```
Comment (gassmoeller) [inline cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md:13]:
```suggestion
Both materials have the same density $\rho=\SI{2800}{\kg\per\cubic\meter}$ and the gravity is vertical with magnitude \SI{9.81}{\meter\per\square\second}.
```

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+The boundary conditions are free slip on the bottom and on the $y=0$ and $y=L_y$ faces,
+free surface at the top, and a velocity $(\pm v_{ext},0,0)$ is prescribed on the
+$x=0$ and $x=L_x$ sides, with $v_{ext}=\SI{0.5}{\cm\per year}$, so as to generate extension in the domain.
+
+Strain weakening is an essential part of this experiment: both cohesion and angle of friction see their values linearly decrease between strains $\varepsilon_1=0.25$ and $\varepsilon_2=1.25$ to reach a maximum weakening factor of $R=C^0/C^{sw}=\phi^0/\phi^{sw}=4$ (i.e. $C^{sw}=\SI{5}{\mega\pascal}$ and $\phi^{sw}=\SI{4}{\degree}$).
+Two weak zones are placed at the base of the upper crust and are characterised by a prescribed strain value of $\varepsilon_2$. As shown in {numref}`fig:allken_setup` they are offset by a distance $\Delta$ which is a multiple (between 2 and 6) of $h_{uc}$.
+The cross section of the two weak zones is $\SI{4}{\km}\times\SI{2}{\km}$ and they are \SI{50}{\km} long.
+
+```{figure-md} fig:allken-results
+<img src="vx.png" style="width:45%" />
```
Comment (gassmoeller) [inline cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md:30]: Unfortunately multiple figures dont work in one figure environment. Either split this into 4 figure environments (that will be shown top to bottom), or merge the figures into one .png or .svg file.

**Review by gassmoeller — COMMENTED** [review]
I am not an expert on these types of setups, it would be great if @naliboff, @anne-glerum, or @MFraters could comment on the setup. I mostly review this to give you feedback on the manual style. Style looks mostly fine, except for the comments I left. You can check the style of the section using the new tester at the bottom of the PR ('Details' will bring you to a rendered version of the documentation using your PR). In your case this is the result of the tester: https://aspect-documentation--4788.org.readthedocs.build/en/4788/user/cookbooks/cookbooks/allken_et_al_2012_rift_interaction/doc/allken.html

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+    set Activation energies for dislocation creep = 0.
+    set Activation volumes for dislocation creep  = 0.
+
+    set Viscosity averaging scheme  = maximum composition
+
+    set Angles of internal friction = 15.
+    set Cohesions                   = 20.e6
+
+    # The parameters below weaken the friction and cohesion by a
+    # a factor of 4 between plastic strain values of 0.5 and 1.5.
```
Comment (anne-glerum) [inline cookbooks/allken_et_al_2012_rift_interaction/allken.prm:141]:
```suggestion
    # a factor of 4 between plastic strain values of 0.5 and 1.25.
```
```suggestion
    # a factor of 4 between plastic strain values of 0.5 and 1.5.
```

Comment (anne-glerum) [inline cookbooks/allken_et_al_2012_rift_interaction/allken.prm:141]:
```suggestion
    # a factor of 4 between plastic strain values of 0.5 and 1.25.
```
```suggestion
    # a factor of 4 between plastic strain values of 0.5 and 1.5.
```

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
+(sec:cookbooks:allken)=
+# Rift interaction in brittle-ductile coupled systems
+
+*This section was contributed by Cedric Thieulot*
+
+The setup for this experiment is based of {cite:t}`alht11,alht12,alhf13`.
```
Comment (anne-glerum) [inline cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md:6]:
```suggestion
The setup for this experiment is based on {cite:t}`alht11,alht12,alhf13`.
```
```suggestion
The setup for this experiment is based of {cite:t}`alht11,alht12,alhf13`.
```

Comment (anne-glerum) [inline cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md:6]:
```suggestion
The setup for this experiment is based on {cite:t}`alht11,alht12,alhf13`.
```
```suggestion
The setup for this experiment is based of {cite:t}`alht11,alht12,alhf13`.
```

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+It is a simple 2-layer crustal model consisting of $h_{uc}=15\text{ km}$ of upper crust (characterised by a Mohr-Coulomb rheology with cohesion
+$C^0=20\text{ MPa}$ and angle of friction $\phi^0=16^o$) and $h_{lc}=15\text{ km}$ of lower crust
+(characterised by a Newtonian viscous rheology with a viscosity $\eta_{lc}=10^{20}\text{ Pa.s}$).
+The flow is assumed to be incompressible and isothermal.
+Both materials have the same density $\rho=2800\text{ kg}/\text{m}^3$ and the gravity is vertical with magnitude $9.81\text{ m}/\text{s}^2$.
+
+```{figure-md} fig:allken-setup
+<img src="allken_setup.png" style="width:80.0%" />
+
+(a) Model setup showing box of dimension. (b) Rheological profile implemented: Mohr-Coulomb plasticity in the upper crust and a fixed viscosity. (c) Frictional plastic strain weakening behavior of the upper crust. Figure taken from {cite}`alht12`.
```
Comment (anne-glerum) [inline cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md:18]: Model setup showing box of dimension....
Something missing from this sentence?

Comment (anne-glerum) [inline cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md:18]: Model setup showing box of dimension....
Something missing from this sentence?

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+The boundary conditions are free slip on the bottom and on the $y=0$ and $y=L_y$ faces,
+free surface at the top, and a velocity $(\pm v_{ext},0,0)$ is prescribed on the
```
Comment (anne-glerum) [inline cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md:22]:
```suggestion
free surface at the top, and an outward velocity $(\pm v_{ext},0,0)$ is prescribed on the
```

Comment (anne-glerum) [inline cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md:22]:
```suggestion
free surface at the top, and an outward velocity $(\pm v_{ext},0,0)$ is prescribed on the
```

**Review by anne-glerum — CHANGES_REQUESTED** [review]
Great to have this example from the literature in! 
The settings for the viscous rheology should be fixed/clarified.

It would be nice to shortly discuss some of the limitations of the setup (or the opportunities for expansion on the setup), such as that it is limited to the crust and constant viscous viscosity, or the results of the study, which looked into the connection of rifts varying with initial distance and overlap of seeds.

**Comment by anne-glerum** [comment]
Sorry for the repeated comments, that just happened, don't know why.

**Comment by anne-glerum** [comment]
Looking at the read the docs test results, there are some issues with the references to figures, some show up as numbers, some as `fig:allken_setup`. Also, some parameter names have super scripts instead of subscripts, like C0, is that intentional?

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 12]
```diff
@@ -0,0 +1,167 @@
...
+  set Initial global refinement          = 2
+  set Time steps between mesh refinement = 1
+  set Refinement fraction                = 0.5
+  set Strategy                           = strain rate
+  set Coarsening fraction                = 0
+end
+
+# Element types
+subsection Discretization
+  set Composition polynomial degree     = 2
```
[the remark this answers is not in the document]
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/allken.prm:45]: done

[the remark this answers is not in the document]
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/allken.prm:45]: done

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 13]
```diff
@@ -0,0 +1,167 @@
...
+  subsection Stokes solver parameters
+    set Number of cheap Stokes solver steps = 0
+  end
+end
+
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 10
```
[the remark this answers is not in the document]
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/allken.prm:24]: done

**Comment by gassmoeller** [comment]
This is ready for another review. @anne-glerum do you have time to take another look?

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 14]
```diff
@@ -0,0 +1,147 @@
+# input file to replicate Allken et al, G3, 2012.
```
Comment (bangerth) [inline cookbooks/allken_et_al_2012_rift_interaction/allken.prm:1]:
```suggestion
# Input file to replicate Allken et al, G3, 2012.
```

**Review by bangerth — COMMENTED** [review]
There are still many comments by @anne-glerum that need to be addressed.

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 15]
```diff
@@ -0,0 +1,167 @@
+#  Global parameters
+set Dimension                              = 3
+set Start time                             = 0
+set End time                               = 5e6
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = iterated Stokes
+set Nonlinear solver tolerance             = 1e-3
```
[the remark this answers is not in the document]
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/allken.prm]: tbh, it was a year ago, but I seem to remember that it was a sound setup when it comes to this issue

[the remark this answers is not in the document]
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/allken.prm]: I just set it to 1e-4 for good measure :)

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 16]
```diff
@@ -0,0 +1,167 @@
...
+  end
+end
+
+# Mesh refinement specifications.
+subsection Mesh refinement
+  set Initial adaptive refinement        = 2
+  set Initial global refinement          = 2
+  set Time steps between mesh refinement = 1
+  set Refinement fraction                = 0.5
+  set Strategy                           = strain rate
```
[the remark this answers is not in the document]
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/allken.prm:40]: I just added the damper params in the prm file

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md (hunk 10 above)
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md]: corrected

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md (hunk 9 above)
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md:6]: done

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md (hunk 11 above)
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md:23]: nice

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm (hunk 14 above)
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/allken.prm:1]: done

Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/allken.prm:1]: sorry @anne-glerum  and @bangerth , I thought I had addressed all comments. I have added the plastic damper params, but I honestly don't feel like testing their influence right now, it is outside the scope of a cookbook I think

**Comment by cedrict** [comment]
@anne-glerum hopefully ok now ?

**Comment by anne-glerum** [comment]
@cedrict Are you sure you pushed your latest changes? My comments have not been marked 'outdated'.

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 17]
```diff
@@ -0,0 +1,167 @@
...
+    set Variable names      = x,y,z
+    set Function constants  = xL=85e3 , xR=135e3
+    set Function expression = 0; \
+                              if( (y<=50e3  && abs(x-xL)<=4e3 && z>=15e3 && z<=19e3) || \
+                                  (y>=160e3 && abs(x-xR)<=4e3 && z>=15e3 && z<=19e3),1.25,0); \
+                              if(z<=15e3, 1, 0);
+  end
+end
+
+# Composition boundary conditions
```
[the remark this answers is not in the document]
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/allken.prm:83]: done

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 18]
```diff
@@ -0,0 +1,167 @@
...
+
+  subsection Visco Plastic
+
+    set Densities                   = 2800
+    set Reference viscosity         = 1.e21
+    set Reference strain rate       = 1.51e-15
+    set Maximum viscosity           = 1.e24
+    set Minimum viscosity           = 1.e19
+
+    set Prefactors for dislocation creep          = 5e-26,1e-50,1e-50, 5e-21
```
[the remark this answers is not in the document]
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/allken.prm:119]: done

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm (hunk 8 above)
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/allken.prm:127]: corrected

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md (hunk 11 above)
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md:23]: corrected

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm (hunk 2 above)
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/allken.prm]: removed

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm (hunk 3 above)
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/allken.prm]: done

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md (hunk 5 above)
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md]: done

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md (hunk 7 above)
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md]: made 4 figures

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md (hunk 9 above)
Comment (cedrict) [reply cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md:6]: done

**Comment by cedrict** [comment]
I hope I have addressed everything

**Comment by gassmoeller** [comment]
The documentation still complains about the missing references. Did you add them to doc/sphinx/references.bib?

**Comment by cedrict** [comment]
@gassmoeller actually the refs were already in the file, under a different 'code' i.e. Allken:etal:2012 etc ... so I used these in my md file

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md (hunk 11 above)
Comment (bangerth) [reply cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md]: As a general rule, there is no need to comment on everything you fix if you fix it the way the reviewer suggested.

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 19]
```diff
@@ -0,0 +1,144 @@
+# Input file to replicate Allken et al, G3, 2012.
+
+#  Global parameters
+set Dimension                              = 3
+set Start time                             = 0
+set End time                               = 0 #5e6
```
Comment (anne-glerum) [inline cookbooks/allken_et_al_2012_rift_interaction/allken.prm:6]:
```suggestion
set End time                               = 5e6
```

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 20]
```diff
@@ -0,0 +1,144 @@
...
+    set Z repetitions = 1
+    set X extent      = 210e3
+    set Y extent      = 210e3
+    set Z extent      = 30e3
+  end
+end
+
+# Mesh refinement specifications.
+subsection Mesh refinement
+  set Initial adaptive refinement        = 1 #2
```
Comment (anne-glerum) [inline cookbooks/allken_et_al_2012_rift_interaction/allken.prm:36]:
```suggestion
  set Initial adaptive refinement        = 2
```

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 21]
```diff
@@ -0,0 +1,144 @@
...
+subsection Mesh deformation
+  set Mesh deformation boundary indicators = top: free surface
+  set Additional tangential mesh velocity boundary indicators = left,right
+  subsection Free surface
+    set Surface velocity projection = normal
+  end
+end
+
+# Velocity boundary conditions
+# The imposed velocity produces a background strain-rate of 2e-15.
```
Comment (anne-glerum) [inline cookbooks/allken_et_al_2012_rift_interaction/allken.prm:53]:
```suggestion
# The imposed velocity produces a background strain-rate of 1.51e-15.
```

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 22]
```diff
@@ -0,0 +1,144 @@
...
+    set Viscous flow law            = diffusion
+    set Grain size exponents for diffusion creep = 0.
+    set Prefactors for diffusion creep          = 5e-26,1e-50,1e-50,5e-21
+    set Activation energies for diffusion creep = 0.
+    set Activation volumes for diffusion creep  = 0.
+    set Viscosity averaging scheme  = maximum composition
+    set Angles of internal friction = 15.
+    set Cohesions                   = 20.e6
+    # The parameters below weaken the friction and cohesion by a
+    # a factor of 4 between plastic strain values of 0.5 and 1.25.
```
Comment (anne-glerum) [inline cookbooks/allken_et_al_2012_rift_interaction/allken.prm:122]:
```suggestion
    # a factor of 4 between plastic strain values of 0.25 and 1.25.
```

File: cookbooks/allken_et_al_2012_rift_interaction/allken.prm [hunk 23]
```diff
@@ -0,0 +1,144 @@
...
+
+# Post processing
+subsection Postprocess
+  set List of postprocessors = basic statistics, composition statistics, temperature statistics, velocity statistics, visualization, topography
+  subsection Visualization
+    set List of output variables = material properties, strain rate, named additional outputs
+    subsection Material properties
+      set List of material properties = density, viscosity
+    end
+    set Time between graphical output = 0e3
```
Comment (anne-glerum) [inline cookbooks/allken_et_al_2012_rift_interaction/allken.prm:141]: Do you need output every timestep? Or can this be every 10 ky for example?

File: cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md [hunk 24]
```diff
@@ -0,0 +1,57 @@
...
+free surface at the top, and an outward velocity $(\pm v_{ext},0,0)$ is prescribed on the
+$x=0$ and $x=L_x$ sides, with $v_{ext}=0.5\text{ cm}/\text{yr}$, so as to generate extension in the domain.
+
+Strain weakening is an essential part of this experiment: both cohesion and angle of friction see their values linearly decrease
+between strains $\varepsilon_1=0.25$ and $\varepsilon_2=1.25$ to reach a maximum weakening factor of $R=C_0/C_{sw}=\phi_0/\phi_{sw}=4$
+(i.e. $C_{sw}=5\text{ MPa}$ and $\phi_{sw}=4^o$).
+Two weak zones are placed at the base of the upper crust and are characterised by a prescribed strain value of $\varepsilon_2$. As shown
+in {numref}`fig:allken-setup` they are offset by a distance $\Delta$ which is a multiple (between 2 and 6) of $h_{uc}$.
+The cross section of the two weak zones is $4\text{ km}\times 2\text{ km}$ and they are $50\text{ km}$ long.
+Results after $1\text{ Myr}$ of extension are shown in {numref}`fig:allken-result1`, {numref}`fig:allken-result2`, {numref}`fig:allken-result3` and {numref}`fig:allken-result4`.
```
Comment (anne-glerum) [inline cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md:33]: Could you add one sentence that describes the main results shown in the figure?

**Review by anne-glerum — CHANGES_REQUESTED** [review]
Hi @cedrict, just a few small comments left! Almost done!

**Review by anne-glerum — APPROVED** [review]
Great, thank you!

**Comment by anne-glerum** [comment]
@gassmoeller Do you want to have another look?

**Review by gassmoeller — APPROVED** [review]
@anne-glerum: I trust your judgment ;-). 
@cedrict: Those are some pretty pictures, put some in the hackathon log.

**Pull request merged**